client.go

```diff
@@ -842,6 +798,53 @@ func (c *TunaSessionClient) removeClosedSessions() {
 	}
 }
 
+func (c *TunaSessionClient) startExits() error {
```

Comment:
You need to make sure tuna are only started once:
- Remove L170-172 to the beginning of this function
- Use lock to protect this function